lms/static/styles/moodle_pages/moodle_pages.scss

```diff
@@ -0,0 +1,249 @@
+// This stylesheet contains a re-implementation of some of Moodle styles for
+// pages, so that Moodle pages served through our LMS app + Via look close to
+// how the page looks in Moodle.
+//
+// The markup generated for pages does not include specific classes, so we only
+// need to define styles for HTML elements.
+//
+// The styles defined here are mainly extracted from the next Moodle files:
+// * Base definition:
+//   * https://github.com/moodle/moodle/blob/main/theme/boost/scss/bootstrap/_reboot.scss
+// * Variables:
+//   * https://github.com/moodle/moodle/blob/main/theme/boost/scss/preset/default.scss
+//   * https://github.com/moodle/moodle/blob/main/theme/boost/scss/bootstrap/_variables.scss
+
+// These are some of the sass variables defined in the files mentioned above:
+$font-family-sans-serif: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+  'Helvetica Neue', Arial, 'Noto Sans', 'Liberation Sans', sans-serif,
+  'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
+$font-family-monospace: SFMono-Regular, Menlo, Monaco, Consolas,
+  'Liberation Mono', 'Courier New', monospace;
+$font-size-base: 0.9375rem;
+$gray-600: #6a737b;
+$gray-900: #1d2125;
+$link-color: #ec7f13;
+$link-hover-color: darken($link-color, 15%);
+
+body {
+  font-family: $font-family-sans-serif;
+  font-size: $font-size-base;
+  line-height: 1.5;
+  color: $gray-900;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin-top: 0;
+  margin-bottom: 0.5rem;
+}
+
+p {
+  margin-top: 0;
+  margin-bottom: 1rem;
+}
+
+ol,
+ul,
+dl {
+  margin-top: 0;
+  margin-bottom: 1rem;
+}
+
+ol ol,
+ul ul,
+ol ul,
+ul ol {
+  margin-bottom: 0;
+}
+
+dt {
+  font-weight: 700;
+}
+
+dd {
+  margin-bottom: 0.5rem;
+  margin-left: 0;
+}
+
+blockquote {
+  margin: 0 0 1rem;
+}
+
+b,
+strong {
+  font-weight: bolder;
+}
+
+small {
+  font-size: 80%;
+}
+
+sub,
+sup {
+  position: relative;
+  font-size: 75%;
+  line-height: 0;
+  vertical-align: baseline;
+}
+
+sub {
+  bottom: -0.25em;
+}
+
+sup {
+  top: -0.5em;
+}
+
+a {
+  color: $link-color;
+  text-decoration: none;
+  background-color: transparent;
+}
+
+a:hover {
+  color: $link-hover-color;
+  text-decoration: underline;
+}
+
+a:not([href]):not([class]) {
+  color: inherit;
+  text-decoration: none;
+}
+
+a:not([href]):not([class]):hover {
+  color: inherit;
+  text-decoration: none;
+}
+
+pre,
+code,
+kbd,
+samp {
+  font-family: $font-family-monospace;
+  font-size: 1em;
+}
+
+pre {
+  margin-top: 0;
+  margin-bottom: 1rem;
+  overflow: auto;
+  -ms-overflow-style: scrollbar;
+}
+
+figure {
+  margin: 0 0 1rem;
+}
+
+img {
+  vertical-align: middle;
+  border-style: none;
+}
+
+svg {
+  overflow: hidden;
+  vertical-align: middle;
+}
+
+table {
+  border-collapse: collapse;
+}
+
+caption {
+  padding-top: 0.75rem;
+  padding-bottom: 0.75rem;
```

Comment:
There are variables for this in the upstream source. It would be helpful to use those here.

Reply:
I have mixed feelings about using too many variables, or in other words, matching too much the non-processed source stylesheets.

I think that might make it harder to evolve in future, as it would make us check multiple source files (which are not very easy to navigate. The structure is relatively complex).

Considering the styles we need are simple, it's more straightforward to compare the actual output/generated CSS and pick the differences from there, if any.

In other words, I think the generated CSS is less likely to change than the source files, that might be refactored, variables renamed, etc, without changing produced CSS, making it harder to find the actual differences in future.